openedx/features/pakx/lms/overrides/views.py

```diff
@@ -451,3 +454,21 @@ class BusinessView(AboutUsView):
     """
     template_name = 'overrides/business.html'
     success_redirect = '/business/#get-started'
+
+    def get_form_initial_data(self, user=None):
+        initial_data = super().get_form_initial_data(user)
+        initial_data.update({'message': 'Not Available. Submitted from Business Page'})
+        return initial_data
+
+
+class MarketingCampaignPage(AboutUsView):
+    """
+    View for business page.
+    """
+    template_name = 'overrides/marketing_campaign.html'
+    success_redirect = '/workplace-harassment/#get-started'
+
+    def get_form_initial_data(self, user=None):
+        initial_data = super().get_form_initial_data(user)
+        initial_data.update({'message': 'Not Available. Submitted from Marketing campaign Page'})
+        return initial_data
```

Comment:
can we introduce a dict `initial_form_data` at the class level which will update the initial_data in `get_form_initial_data`?